src/Aggregation/Composite.php

```diff
@@ -25,6 +25,10 @@ public function addSource(AbstractAggregation $aggregation): self
      */
     public function addAfter(?array $checkpoint): self
```

Comment:
Cannot we deprecate passing null? So that we can change the type-hint in 8.0 to only allow array and remove the check for null?

Reply:
I'd be in favor of keeping it, It's very handy for writing code:

```
do {
    $result = $this->doYourAggregation($after ?? null);

    // ...

    $after = $result->getAggregations()['composite']['after_key'];
} while (true);
```